example/src/App.js

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react'
 
-import ExampleComponent from 'react-barong'
+import LoginApp from 'react-barong'
 
 export default class App extends Component {
   render () {
     return (
       <div>
-        <ExampleComponent text='Modern React component module' />
+        <LoginApp />
```

Comment:
wrong usage
LoginApp is an ExampleApp importing and using those two component

        <LoginForm />
￼        <RegisterForm />